packages/@okta/vuepress-site/docs/guides/oie-embedded-common-refresh-tokens/main/android/refreshusingthesdk.md

```diff
@@ -0,0 +1,5 @@
+## Option 1: Refresh using the Android SDK
+
+The [Android SDK, Widget and sample app](https://github.com/okta/okta-idx-android)
```

Comment:
```suggestion
The [Android SDK, Widget, and sample app](https://github.com/okta/okta-idx-android)
```

Reply:
Hi Susan, is removing a comma before the second to last item in a list, correct? Unless I'm intepretting it wrong, the Okta style guide and Microsoft style guide both use a comma before the second to the last item in a list of three or more items.

**Okta style guide linke:** https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide#3.-Best-practices-for-common-mistakes

**Microsoft style guide link:**
https://docs.microsoft.com/en-us/style-guide/punctuation/commas

Reply:
I'm adding a comma :)

Reply:
Yes, now i see - thanks for the clarification. Will make the update.